app/src/main/java/org/covidwatch/android/data/contactevent/ContactEventsDownloadWorker.kt

```diff
@@ -8,11 +8,17 @@ import org.covidwatch.android.R
 import org.covidwatch.android.data.ContactEventDAO
 import org.covidwatch.android.data.CovidWatchDatabase
 import org.covidwatch.android.data.contactevent.firebase.FirebaseContactEventFetcher
+import org.covidwatch.android.domain.NotifyAboutPossibleExposureUseCase
+import org.koin.core.KoinComponent
+import org.koin.core.inject
 import java.util.*
 import java.util.concurrent.TimeUnit
 
-class ContactEventsDownloadWorker(private val context: Context, workerParams: WorkerParameters) :
-    Worker(context, workerParams) {
+class ContactEventsDownloadWorker(
+    private val context: Context,
+    workerParams: WorkerParameters
+) : Worker(context, workerParams),
+    KoinComponent {
```

Comment:
Do you need to add this to add to the `DI` chain so it can be injected? Or just allow it to use injectable things

Reply:
Yep, from docs:
>KoinComponent interface marker to bring Koin extensions features

However koin has already marked most of the android components (Application, Activity, Fragment) so we don't to mark them manually